### DISCUSSION TITLE
Visualizaton of nodes in grid

### DISCUSSION THREAD

**Discussion by dgmyr** [Security-Onion-Solutions/securityonion#9030] [Unsupported Versions] [answered]
Hello, I'm trying to hide all nodes in grid page in the SOC Console using an example in /opt/so/saltstack/default/salt/soc/files/soc/custom_roles but all nodes still visible for a custom user that don't have a permissions to see them.

**Comment by dougburks** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/discussions/8991#discussioncomment-4028039
